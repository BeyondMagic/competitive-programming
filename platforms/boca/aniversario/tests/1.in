2
Natalia 28
Wilson 82
