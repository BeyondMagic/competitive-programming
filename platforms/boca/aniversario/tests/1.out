Em 0 anos Natalia faz 28 e Wilson 82
Em 11 anos Natalia faz 39 e Wilson 93
